bot/code_review_bot/workflow.py

```diff
@@ -224,9 +224,18 @@ def _in_group(dep_id):
             try:
                 if isinstance(dep, type) and issubclass(dep, AnalysisTask):
                     # Build a class instance from its definition and route
-                    task = dep.build_from_route(self.index_service, self.queue_service)
```

Comment:
`dep.build_from_route` should never crash: the patch would be way cleaner by catching the exception there and adding a log in the `if` case here. 
I don't like having to check on the ZeroCoverageTask , that's not generic & we could have that kind of failures on other parts

Reply:
Since this is a very weird failure, I wanted to keep raising an exception unless strictly necessary and only log for the already known failure.
I don't mind logging always though, so I'll just do that.